### PR TITLE
Added rebootCommand values

### PR DESCRIPTION
**Pull request by renaudhager**
Hi all,

This PR adds `--reboot-command` flag in the chart.

Let me know if anything needs to be changed.

### CODE REVIEW

**Comment by ckotzbauer** [comment]
Thanks for your contribution @renaudhager.
Could you please increase the `version` in the `Chart.yaml` file from `2.6.0` to `2.7.0` as its a feature-addition. This also should fix the failing Github-Check for testing.

**Comment by renaudhager** [comment]
> Thanks for your contribution @renaudhager.
> Could you please increase the `version` in the `Chart.yaml` file from `2.6.0` to `2.7.0` as its a feature-addition. This also should fix the failing Github-Check for testing.

Done.

**Comment by renaudhager** [comment]
Some checks seem unhappy, but I don't understand why. Let me know what I need to change to fix them.

File: charts/kured/values.yaml [hunk 1]
```diff
@@ -31,6 +31,7 @@ configuration:
   prometheusUrl: ""          # Prometheus instance to probe for active alerts
   rebootDays: []             # only reboot on these days (default [su,mo,tu,we,th,fr,sa])
   rebootSentinel: ""         # path to file whose existence signals need to reboot (default "/var/run/reboot-required")
+  rebootCommand: "/sbin/systemctl reboot" #command to run when a reboot is required by the sentinel
```
Comment (ckotzbauer) [inline charts/kured/values.yaml:34]: This `# command` should to the trick instead of `#command`

**Comment by ckotzbauer** [comment]
Could you please add the new param to the README of the chart? If all checks are green then, this should be ready 😉

**Comment by renaudhager** [comment]
> Could you please add the new param to the README of the chart? If all checks are green then, this should be ready 😉

Done, hopefully it should all good this time :-)

**Comment by ckotzbauer** [comment]
I'll have a look later today what's wrong here.

**Review by ckotzbauer — APPROVED** [review]

**Comment by alexeldeib** [comment]
@renaudhager I noticed in this PR, the chart value is set differently from the in-code default used by Kured. Was that deliberate? It breaks the default chart deployment on platforms like AKS, ref https://github.com/weaveworks/kured/issues/404

**Comment by jackfrancis** [comment]
@alexeldeib good find!

https://github.com/weaveworks/kured/pull/408

**Pull request merged**